fvm/fvm.go

```diff
@@ -45,25 +47,34 @@ func NewVirtualMachine(rt runtime.Runtime) *VirtualMachine {
 
 // Run runs a procedure against a ledger in the given context.
 func (vm *VirtualMachine) Run(ctx Context, proc Procedure, v state.View, programs *programs.Programs) (err error) {
-	meterParams, err := getEnvironmentMeterParameters(
+	meterParams := meter.DefaultParameters().
+		WithComputationLimit(uint(proc.ComputationLimit(ctx))).
```

Comment:
Question, for service account, do we also need to disable computation limit?

Reply:
I've asked the same question (I personally would have prefer to disable the computation limit).

The answer I got is, since we can specify gas limit (but not memory/interaction limits) as part of the transaction, we should maintain the computation limit and just set the gas limit to max.  This enables other cadence users to use service account transactions as example when they develop their own transaction.